app/assets/stylesheets/application.scss

```diff
@@ -34,23 +38,116 @@ body {
 
 .enter-question {
   padding: 0;
+  background-color: #000DBA;
+  border-color: #007095;
+  color: #6FFDFF;
 }
 
 .main-content-container {
   padding: 0 15px;
 }
 
-input[type=text].postfix.round,
-input[type=email].postfix.round {
-  border-radius: 1000px 0 0 1000px;
-  padding: 0;
+.background {
+  display: flex;
```

Comment:
Properties should be ordered background, background-position, -moz-background-size, -o-background-size, -webkit-background-size, background-size, display, flex-flow, height, margin-left, margin-right, max-width, overflow